src/pages/workspaces/workspace/analysis/tool-utils.ts

```diff
@@ -46,34 +47,44 @@ export interface AppTool extends Tool {
   appType: string
 }
 
+const RStudio: RuntimeTool = { label: 'RStudio', ext: ['Rmd', 'R'] as Extension[], imageIds: ['RStudio'], defaultImageId: 'RStudio', defaultExt: 'Rmd' as Extension }
```

Comment:
would it be valuable to tie the `label` field here to the value of `toolLabels[SomeTool]` (or vice versa), so that the value is defined only once?

I'm also seeing these labels used more like enums or constants (fixed values representing choices) rather than as what I'd think of as "labels" (human readable). But the definition of a term like "label" is very fluid and if a "tool label" in this context feels more like an immutable (or at least rarely mutable, over the life of the project) tag, then no action needed.